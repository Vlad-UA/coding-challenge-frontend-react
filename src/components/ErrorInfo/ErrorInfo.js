// Core
import React, { PureComponent } from 'react';

// Instruments
import styled from 'styled-components';

const Message = styled.p`color: red;`;

export default class ErrorInfo extends PureComponent {
  render() {
    return (
      <div>
        <Message>Ooops, something went wrong</Message>
      </div>
    );
  }
}
