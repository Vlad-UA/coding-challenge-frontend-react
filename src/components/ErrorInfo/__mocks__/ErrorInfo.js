// Core
import React from 'react';

const ErrorInfoMock = () => <div className="error-info-mock">ErrorInfoMock</div>;
ErrorInfoMock.displayName = 'ErrorInfoMock';
export default ErrorInfoMock;
