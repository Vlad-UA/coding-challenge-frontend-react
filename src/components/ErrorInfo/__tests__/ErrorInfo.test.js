// Core
import React from 'react';

// Test utilities
import renderer from 'react-test-renderer';

// Component
import ErrorInfo from '../ErrorInfo';

describe('Component ErrorInfo:', () => {
  test('Must match snapshot:', () => {
    const errorInfo = renderer
      .create(<ErrorInfo />)
      .toJSON();

    expect(errorInfo).toMatchSnapshot();
  });
});
