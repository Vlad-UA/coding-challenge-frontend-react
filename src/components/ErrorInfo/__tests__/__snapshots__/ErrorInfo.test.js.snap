// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component ErrorInfo: Must match snapshot: 1`] = `
<div>
  <p
    className="sc-bdVaJa erWExG"
  >
    Ooops, something went wrong
  </p>
</div>
`;
